src/Ares.php

```diff
@@ -300,6 +293,7 @@ public function findByName($name, $city = null)
 
         $records = new AresRecords();
         foreach ($elements as $element) {
+            // TODO: What is this?
             $record = new AresRecord();
             $record->setCompanyId(strval($element->ico));
             $record->setTaxId(
```

Comment:
@dfridrich please, could you give me a little advice, why should be here ares record incomplete?